fabric-model-loading-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/model/loading/ModelTestModClient.java

```diff
@@ -34,6 +41,15 @@ public class ModelTestModClient implements ClientModInitializer {
 	public void onInitializeClient() {
 		ModelLoadingPlugin.register(pluginContext -> {
 			pluginContext.addModel(MODEL_ID);
+			pluginContext.onBakedModelLoad().register(context -> {
+				if(context.location().getPath().equals("block/dirt")) {
```

Comment:
This is nasty and better achieved using a `ForwardingBakedModel` - I can fix this up later myself once this is in my branch, no worries.

Reply:
I would never do this in production as it would outright break with something like FerriteCore present, I think. Was only here because this is a test mod, and I didn't know `ForwardingBakedModel` existed.

Reply:
Cleaned this up, it's actually simpler with the ForwardingBakedModel anyway.